integration-tests/src/query_params.test.js

```diff
@@ -91,6 +91,66 @@ describe("for a GET endpoint with no parameters", () => {
       last: "Last"
     });
   });
+
+  test("server will parse empty inputs on non optional parameters as empty strings", async () => {
+    const server = TestUtils.makeServer({
+      endpoint: testEndpoint,
+      handler: testHandler
+    });
+    const resp = await fetch(
+      `http://localhost:${server.address().port}/foo?first=&last=Last`
+    );
+    await resp.json();
+    expect(testHandler).toHaveBeenLastCalledWith({
+      first: "",
+      last: "Last"
+    });
+  });
+});
+
+const testNumEndpoint: Recouple.Endpoint<
+  {
+    x: number
+  },
+  number
+> = Recouple.endpoint()
+  .fragment("foo")
+  .queryParams({
+    x: T.number
+  });
+
+const testNumHandler = jest.fn(async () => {
+  return 0;
+});
+
+describe("for a GET endpoint with number query parameters", () => {
+  let server;
+  beforeEach(() => {
+    server = TestUtils.makeServer({
+      endpoint: testNumEndpoint,
+      handler: testNumHandler
+    });
+  });
+
+  describe("for the recouple client", () => {
+    it("can be called with a query parameter", async () => {
+      const baseURL = `http://localhost:${server.address().port}`;
+      const input = { x: 47 };
+      await RecoupleFetch.safeGet(baseURL, testNumEndpoint, input);
+      const expectedURL = `${baseURL}/foo?x=47`;
```

Comment:
this could be directly put into `toHaveBeenLastCalledWith`, but not necessary

Reply:
Yeah... We should stay consistent, but I think this is more readable as long as these tests remain short.